### PR TITLE
Issue 16939 & 17981 related content dialog checkbox issues

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

**Review by hmoreras — APPROVED** [review]

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 1]
```diff
@@ -1,74 +1,160 @@
...
+    />
+    <input type="hidden" dojoAttachPoint="tagText" value="<%=
+    LanguageUtil.get(pageContext,
+    "Type-your-tag-You-can-enter-multiple-comma-separated-tags") %>"> <input
+    type="hidden" dojoAttachPoint="suggestedTagsText" value="<%=
+    LanguageUtil.get(pageContext, "Suggested-Tags") %>"> <input type="hidden"
+    dojoAttachPoint="noResultsText" value="<%= LanguageUtil.get(pageContext,
+    "No-Results-Found") %>"> <input type="hidden"
+    dojoAttachPoint="matchResultsText" value="<%= LanguageUtil.get(pageContext,
+    "Matching-Results") %>">
```
Comment (freddyucv) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: it is hard to see that we have multi inputs hidden tag here, maybe you should split them into several lines

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 2]
```diff
@@ -1,74 +1,160 @@
...
-		   </div>
-	
-	    </div>
-
-</div>
-    </form>
-
+    </div>
+    <!-- START Right Column -->
+    <div dojoType="dijit.layout.ContentPane" class="portlet-main-wrapper">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 3]
```diff
@@ -1,74 +1,160 @@
...
+                    iconClass="previousIcon"
+                >
+                    <%= LanguageUtil.get(pageContext, "Previous") %>
+                </button>
+            </div>
+            <div></div>
+            <div dojoAttachPoint="nextDiv" style="display: none;">
+                <button
+                    dojoType="dijit.form.Button"
+                    class="bg"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 4]
```diff
@@ -1,74 +1,160 @@
...
+                    dojoAttachPoint="matchingResultsDiv"
+                    class="portlet-toolbar__matching-results"
+                    style="visibility: hidden"
+                >
+                    <%= LanguageUtil.get(pageContext, "Results") %>
+                </div>
+                <div id="addContentTypeDropdown"></div>
+                <div
+                    dojoAttachPoint="addContentletButton"
+                    class="portlet-toolbar__add-contentlet"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 5]
```diff
@@ -1,74 +1,160 @@
...
+                dojoAttachPoint="structure_inode"
+                value="strInode"
+            />
+            <div class="sideMenuWrapper">
+                <div dojoAttachPoint="structures_select"></div>
+                <div dojoAttachPoint="search_languages_table"></div>
+                <div dojoAttachPoint="search_fields_table"></div>
+                <div dojoAttachPoint="search_categories_table">
+                    <dl
+                        class="vertical"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 6]
```diff
@@ -1,74 +1,160 @@
...
-                        <button dojoType="dijit.form.Button" dojoAttachEvent='onClick:_clearSearch' iconClass="cancelIcon" class="dijitButtonFlat" style="margin-top: 16px"><%= LanguageUtil.get(pageContext, "Clear") %></button>
-                    </div>
+<form
+    dojoAttachPoint="search_form"
+    onsubmit="return false;"
+    id="searchForm"
+    class="related-content-form"
+>
+    <!-- START Left Column -->
+    <div dojoType="dijit.layout.ContentPane" class="portlet-sidebar-wrapper">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 7]
```diff
@@ -1,74 +1,160 @@
...
-                    <div class="clear"></div>
-                    <div class="buttonRow">
-                        <button dojoType="dijit.form.Button" dojoAttachEvent='onClick:_doSearchPage1' iconClass="searchIcon"><%= LanguageUtil.get(pageContext, "Search") %></button>
-                        <button dojoType="dijit.form.Button" dojoAttachEvent='onClick:_clearSearch' iconClass="cancelIcon" class="dijitButtonFlat" style="margin-top: 16px"><%= LanguageUtil.get(pageContext, "Clear") %></button>
-                    </div>
+<form
+    dojoAttachPoint="search_form"
+    onsubmit="return false;"
+    id="searchForm"
+    class="related-content-form"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 8]
```diff
@@ -1,74 +1,160 @@
...
+                        dojoAttachEvent="onClick:_doRelateContent"
+                        iconClass="searchIcon"
+                    >
+                        <%= LanguageUtil.get(pageContext, "Relate") %>
+                    </button>
+                </div>
+            </div>
+            <table
+                dojoAttachPoint="results_table"
+                class="listingTable relateContent"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 9]
```diff
@@ -1,74 +1,160 @@
...
+                        dojoAttachEvent="onClick:_doSearchPage1"
+                        iconClass="searchIcon"
+                    >
+                        <%= LanguageUtil.get(pageContext, "Search") %>
+                    </button>
+                    <button
+                        dojoType="dijit.form.Button"
+                        dojoAttachEvent="onClick:_clearSearch"
+                        iconClass="cancelIcon"
+                        class="dijitButtonFlat"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 10]
```diff
@@ -1,74 +1,160 @@
...
-    </form>
-
+    </div>
+    <!-- START Right Column -->
+    <div dojoType="dijit.layout.ContentPane" class="portlet-main-wrapper">
+        <div dojoAttachPoint="contentWrapper">
+            <div class="portlet-toolbar">
+                <div
+                    dojoAttachPoint="matchingResultsDiv"
+                    class="portlet-toolbar__matching-results"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp (hunk 1 above)
Comment (fmontes) [reply dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: +1

**Review by fmontes — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**